# oteodoro changes:
# Patch status:  testing
# Append hardened Full RELRO to -v for ebuilds that require matching security
# For use for the hardened profile
diff -urp lld.orig/ELF/Driver.cpp lld/ELF/Driver.cpp
--- lld.orig/ELF/Driver.cpp	2021-09-29 01:49:47.555537727 -0700
+++ lld/ELF/Driver.cpp	2021-09-29 01:57:55.618529914 -0700
@@ -500,8 +500,10 @@ void LinkerDriver::linkerMain(ArrayRef<c
   // lot of "configure" scripts out there that are generated by old version
   // of Libtool. We cannot convince every software developer to migrate to
   // the latest version and re-generate scripts. So we have this hack.
-  if (args.hasArg(OPT_v) || args.hasArg(OPT_version))
+  if (args.hasArg(OPT_v) || args.hasArg(OPT_version)) {		// Changed by oteodoro
     message(getLLDVersion() + " (compatible with GNU linkers)");
+    message("Hardened:  Full RELRO");					// Added by oteodoro for eclasses/ebuilds to check
+  }									// Added by oteodoro
 
   if (const char *path = getReproduceOption(args)) {
     // Note that --reproduce is a debug option so you can ignore it
